actions/mod_info_MOD.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   name: 'Welcome',
-  section: '#Mod Information',
+  section: '#DBM Mods',
```

Comment:
This should be updated in docs file

Reply:
Also seems like there are some links pointing to the old repository URL. Should probably update those